src/com/homework/MyArrayList.java

```diff
@@ -0,0 +1,50 @@
+package com.homework;
+
+import java.util.Arrays;
+
+public class MyArrayList<E> implements MyList<E> {
+    private final int DEFAULT_CAPACITY = 10;
+    transient Object[] elementData;
```

Comment:
Why 'transient'? Explain

Reply:
spied it in sources of ArrayList - there this modifier was explained as follows: "non-private to simplify nested class access"  - so i decided to leave it as is

Reply:
Please change it. Don't copy all from sources)

Reply:
changed it to private  - in MyStack and MyArrayList